### PR TITLE
hobbes: fix build error

### PR DESCRIPTION
**Pull request by sophrosyne97**
###### Motivation for this change
ZHF: #122042
cc @NixOS/nixos-release-managers
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/hobbes/default.nix [hunk 1]
```diff
@@ -20,8 +20,10 @@ stdenv.mkDerivation {
     ncurses
     readline
     zlib
+    libxml2
   ];
 
+
```
Comment (jonringer) [inline pkgs/development/tools/hobbes/default.nix:26]: extra newline
```suggestion
```

**Review by jonringer — COMMENTED** [review]
otherwise diff LGTM

File: pkgs/development/tools/hobbes/default.nix (hunk 1 above)
Comment (sophrosyne97) [reply pkgs/development/tools/hobbes/default.nix]: fixed

**Review by risicle — APPROVED** [review]
WFM macos 10.15, nixos x86_64

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123944` at a156f033 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>hobbes</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/name-and-version.md">name-and-version</a>

Did you mean to pass `pname` instead of `name` to `mkDerivation`?

Near pkgs/development/tools/hobbes/default.nix:4:3:
```
  |
4 |   name = "hobbes";
  |   ^
```

Near pkgs/development/tools/hobbes/default.nix:5:3:
```
  |
5 |   version = "unstable-2020-05-19";
  |   ^
```
</li>
  </ul>
</details>

**Review by thmzlt — APPROVED** [review]

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 123944 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  <ul>    <li>hobbes</li>  </ul></details>

**Pull request merged**